client/src/components/jobPosting/JobPostingForm.js

```diff
@@ -29,7 +29,7 @@ export class JobPostingForm extends Component {
 
   componentDidMount() {
     if (this.state.mode === 'edit') {
-      const jobPostingId = window.location.href.split('/')[4]
+      const jobPostingId = this.props.match.params.id
```

Comment:
For consistency's sake, this should be in the same format as the others. There might be more tests added later on